docs/source/examples/docker-containers.rst

```diff
@@ -3,16 +3,36 @@
 Using Docker Containers
 =======================
 
-SkyPilot can run a Docker container either as the runtime environment for your task, or as the task itself.
+SkyPilot can run a Docker container either as a containerized application, or the runtime environment for your task.
+
+Running Containerized Application
+---------------------------------
+
+SkyPilot is capable of running containerized application directly. The Docker runtime comes pre-configured and ready for use on the default VM image provided by SkyPilot, making it convenient for executing containerized applications.
+
+To launch a containerized application, you can directly invoke the :code:`docker run` command in the :code:`run` section of your task.
+
+For example, to run a hugging face TGI serving container:
+
+.. code-block:: yaml
+
+  resources:
+    accelerators: A100:1
+
+  run: |
+    docker run --gpus all --shm-size 1g -v ~/data:/data ghcr.io/huggingface/text-generation-inference --model-id lmsys/vicuna-13b-v1.5
 
 Using Docker Containers as Runtime Environment
 ----------------------------------------------
 
-When a container is used as the runtime environment, the SkyPilot task is executed inside the container.
+When a container is used as the runtime environment, all the SkyPilot task is executed inside the container.
+
+This means all :code:`setup` and :code:`run` commands in the YAML file will be executed in the container, and any files created by the task will be stored inside the container. The Docker container functions as an image ID within the virtual machines provided by cloud service providers.
 
-This means all :code:`setup` and :code:`run` commands in the YAML file will be executed in the container, and any files created by the task will be stored inside the container.
 Any GPUs assigned to the task will be automatically mapped to your Docker container and all future tasks on the cluster will also execute in the container.
```

Comment:
A bit confused by this sentence. There's 1 container per node of the cluster, right? So "all future tasks on the cluster will also execute in the container." seems inaccurate in the multinode case (there's no single container but N containers). How to best rephrase?

Reply:
Now changed to:
```text
Any GPUs assigned to the task will be automatically mapped to your Docker container, and all subsequent tasks within the cluster will also run inside the container. In a multi-node scenario, the container will be launched on all nodes, and the corresponding node's container will be assigned for task execution.
```